packages/orchestration/test/examples/stake-atom.contract.test.ts

```diff
@@ -0,0 +1,65 @@
+import { test } from '@agoric/zoe/tools/prepare-test-env-ava.js';
+
+import { AmountMath } from '@agoric/ertp';
+import { setUpZoeForTest } from '@agoric/zoe/tools/setup-zoe.js';
+import { E } from '@endo/far';
+import path from 'path';
+import type { Installation } from '@agoric/zoe/src/zoeService/utils.js';
+import { commonSetup } from '../supports.js';
+
+const dirname = path.dirname(new URL(import.meta.url).pathname);
+
+const contractFile = `${dirname}/../../src/examples/stakeAtom.contract.js`;
+type StartFn =
+  typeof import('@agoric/orchestration/src/examples/stakeAtom.contract.js').start;
+
+const coreEval = async (
+  t,
+  { orchestration, timer, marshaller, storage, bld },
+) => {
+  t.log('install stakeAtom contract');
+  const { zoe, bundleAndInstall } = await setUpZoeForTest();
```

Comment:
it's awkward to create zoe in the core-eval for starting the contract

Reply:
obsolete name after refactoring. I'll change them all.